R/auto.R

```diff
@@ -414,7 +414,7 @@ create_pull_request <- function(release_branch, main_branch, remote_name, force)
     # Remove cached config so that pr_url() always checks
     # if we happened to overwrite the branch
     config_url <- glue("branch.{release_branch}.pr-url")
-    rlang::exec(git2r::config, !!config_url := NULL)
+    rlang::exec(gert::git_config_set, !!config_url, NULL)
```

Comment:
Do we even need tidy evaluation here?

```suggestion
    gert::git_config_set(config_url, NULL)
```

Reply:
I don't know, did not test in detail. Let's try without.